validator/client/src/main/java/tech/pegasys/teku/validator/client/KeyManager.java

```diff
@@ -92,8 +104,46 @@ public DeleteKeysResponse deleteValidators(final List<BLSPublicKey> validators)
    * @param slashingProtection a combined slashing protection payload
    * @return a list of 1 status per keystore that was attempted to be imported
    */
-  public List<ImportStatus> importValidators(
+  public List<PostKeyResult> importValidators(
       final List<String> keystores, final List<String> passwords, final String slashingProtection) {
-    throw new NotImplementedException("importValidators not implemented yet");
+
+    final List<PostKeyResult> postKeyResults = new ArrayList<>();
+
+    if (keystores.size() == passwords.size()) {
+      final Iterator<String> keystoreIterator = keystores.iterator();
+      final Iterator<String> passwordIterator = passwords.iterator();
+      while (keystoreIterator.hasNext() && passwordIterator.hasNext()) {
```

Comment:
nit: Given we have two `List` instances and have checked the size, I would typically use an old-school `for (int i = 0; i < keystores.size(); i++) {` type loop rather than using iterators. Mostly because it winds up being shorter but also because then we're consistently using `.size()` in the checks so they are more obviously tied together.